swap/src/bin/swap_cli.rs

```diff
@@ -112,14 +112,29 @@ async fn main() -> Result<()> {
             let monero_wallet =
                 init_monero_wallet(monero_network, monero_wallet_rpc_process.endpoint()).await?;
             let bitcoin_wallet = Arc::new(bitcoin_wallet);
+            let (event_loop, mut event_loop_handle) = EventLoop::new(
+                &seed.derive_libp2p_identity(),
+                alice_peer_id,
+                alice_addr,
+                bitcoin_wallet.clone(),
+            )?;
+            let handle = tokio::spawn(event_loop.run());
+
+            debug!("Requesting quote");
+
+            let bid_quote = event_loop_handle
+                .request_quote()
+                .await
+                .context("failed to request quote")?;
 
-            let swap_id = Uuid::new_v4();
+            info!("Received quote: 1 XMR = {}", bid_quote.price);
```

Comment:
```suggestion
            info!("The current price is 1 XMR = {} (the price during the swap may differ)", bid_quote.price);
```

This message is correct, but we should probably deliver that information differently, because the user has to understand that this quote is not equivalent to the amount that will be exchanged during the swap. It is a quote at the given time, where the swap will use the price at the point where the swap is started.

Reply:
Yep, will update the message!